metadata/demographic/census_blocks_2020.md

```diff
@@ -1,90 +1,77 @@
 
 ### How reliable and accurate is the dataset
 
-The 2020 data available on this page was downloaded from the [2020 Census Redistricting Data (P.L. 94-171)](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2). All 2020 census geographies reflect boundaries as of January 1, 2020. Additional GIS files and support products are available on the [census redistricting data](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2).
-
-These datasets were downloaded from the 2020 Census Redistricting Data (P.L. 94-171) Page. All 2020 Census boundaries are current as of January 1, 2020. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, AGRC will append the demographic data to the existing 2020 geographies found on this page.
+The data available on this page was downloaded from the [2020 Census Redistricting Data (P.L. 94-171)](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2). All 2020 census geographies reflect boundaries as of January 1, 2020. Additional GIS files and support products are available on the [census redistricting data](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2).
 
 ## Credits
 
 ### Data Source
 
-<!-- Who created or provides the data. For roads, this would be UDOT, Counties -->
 US Census Bureau
 
 ### Host
 
-<!-- Who is aggregating and hosting the feature service, downloadable zip file, etc. For roads, this would be UGRC -->
 UGRC
 
 ## Restrictions
 
-<!-- Any usage limitations or constraints on where or how the dataset can be used, including disclaimers and attribution rules -->
-
 For more information on usage limitations for this dataset, please see our disclaimer [here](https://gis.utah.gov/documentation/policy/license/#disclaimer).
 
 ## License
-<!-- The license the data are released under. Will usually be CC BY 4.0, but could be different. -->
+
 CC BY 4.0 License
 
 ## Tags
 
-<!-- Each data set's tags should include the stewarding agency (UGRC, DWR, etc), "SGID," and the layer's category. Add any other relevant tags, but don't include any words in the layer's title-->
-UGRC, SGID, US Census Bureau
+- UGRC
+- SGID
+- US Census Bureau
 
 ## Secondary Category
 
-<!-- Another category the layer fits in, if applicable. Can be left blank. -->
-
 ## Data Page Link
 
-<!-- Link to the layer's data page on gis.utah.gov -->
 https://gis.utah.gov/products/sgid/demographic/census-2020-blocks/
 
 ## Update
```

Comment:
@jacobdadams if there not going to be text under this heading, we should probably remove it from the template

Reply:
Yeah, I guess that's just in there for hierarchy's sake. Should we do the same with the description heading, and move all the level 3 items to level 2

Reply:
yeah that makes sense to me. it will make things a little easier in the future